test/config/jds-do-not-panic-if-jdc-close-connection/jds-config.toml

```diff
@@ -1,20 +0,0 @@
-# SRI Pool config
-authority_public_key = "9auqWEzQDVyd2oe1JVGFLMLHZtCo2FFqZwtKA5gd9xbuEu7PH72"
-authority_secret_key = "mkDLTBBRxdBv998612qipDYoTK3YUrqLe8uWw7gu3iXbSrn2n"
```

Comment:
IMO deleting the MG files and CI appearances of the same test should be in the same commit, so I would combine this with the one before it